React/Base/RCTRootView.m

```diff
@@ -168,7 +168,7 @@ - (RCTModuleRegistry *)moduleRegistry
 #if TARGET_OS_TV
 - (NSArray<id<UIFocusEnvironment>> *)preferredFocusEnvironments
 {
-  if (self.reactPreferredFocusedView) {
+  if (self.reactPreferredFocusedView && self.reactPreferredFocusedView.window != nil) {
```

Comment:
Unfortunately, for short moment after view unmount, this reference is still not released, therefore additional check is needed.

Reply:
This need to be verified in case of some custom views, like modals, maybe it makes sense for unattached view, to be preferredFocusEnvironment in some cases?